pkgs/top-level/all-packages.nix

```diff
@@ -7935,7 +7935,7 @@ with pkgs;
 
   noip = callPackage ../tools/networking/noip { };
 
-  nomad = nomad_1_1;
+  nomad = nomad_1_2;
```

Comment:
I usually would wait until .3 .4 or so before doing this - early nomad builds tend to be a liiitle risky to adopt

Reply:
I tend to agree, though now that i'm looking at it, I don't think the 1.2 point release is like their previous point releases. There are security changes in 1.2 (See 1.2.1 and 1.2.2 (release notes)[https://github.com/hashicorp/nomad/releases] that have not been backported to 1.1.x which makes me think that 1.1 is not gonna get the long term support like we saw with 0.12.

Reply:
The 1.2.2 bug fix was for a bug introduced in 1.2.x, and the 1.2.1 CVE was also fixed in 1.1.8, but didn't get a GH release (just a tag/all the other stuff)

Reply:
Can you decide on something here? If the merge conflict is resolved we can merge this.